grafanimate/grafana-studio.js

```diff
@@ -139,14 +104,17 @@ class GrafanaStudioSrv {
       }
       var url = "/" + view + "/" + uid + "/" + slug + query;
 
+      // TODO We need a replacement for Grafana 11 for this
+      // which opens a dashboard without a reload
+
```

Comment:
@intermittentnrg do you have any idea for this - maybe from slack?

There once was something like `$location.url(url);` to update the url

We could use `window.location.assign(url)` - however we then lose the current JS context and our grafana-studio is not available anymore.
This is not crucial, I think most parts are working now :)

Reply:
I don't. I only had 2 questions for them:
1. set time range.
2. confirm loading&rendering completed.

As I said in another reply I've never specified a different dashboard uid than is already in URL.
Feel free to join grafana slack and ask, they seem to be more responsive there than on GH issues.